src/command_modules/azure-cli-eventgrid/azure/cli/command_modules/eventgrid/commands.py

```diff
@@ -32,12 +46,30 @@ def load_command_table(self, _):
                                  setter_name='update',
                                  client_factory=topics_factory)
 
+    with self.command_group('eventgrid domain topic', domain_topics_mgmt_util, client_factory=domain_topics_factory) as g:
+        g.command('show', 'get')
+        g.custom_command('list', 'cli_domain_topic_list')
+        g.custom_command('delete', 'cli_domain_topic_delete')
+        g.custom_command('create', 'cli_domain_topic_create_or_update')
+
+    with self.command_group('eventgrid domain', domains_mgmt_util, client_factory=domains_factory) as g:
+        g.command('show', 'get')
+        g.command('key list', 'list_shared_access_keys')
+        g.command('key regenerate', 'regenerate_key')
+        g.custom_command('list', 'cli_domain_list')
+        g.custom_command('create', 'cli_domain_create_or_update')
+        g.command('delete', 'delete')
+        g.generic_update_command('update',
+                                 getter_name='get',
+                                 setter_name='update',
+                                 client_factory=domains_factory)
+
     custom_tmpl = 'azure.cli.command_modules.eventgrid.custom#{}'
     eventgrid_custom = CliCommandType(operations_tmpl=custom_tmpl)
 
     with self.command_group('eventgrid event-subscription', client_factory=event_subscriptions_factory) as g:
         g.custom_command('create', 'cli_eventgrid_event_subscription_create')
-        g.custom_show_command('show', 'cli_eventgrid_event_subscription_get')
+        g.custom_command('show', 'cli_eventgrid_event_subscription_get')
```

Comment:
Should use custom_show_command. Why was this changed?

Reply:
Fixed